src/app/data/schedule-info.js

```diff
@@ -16,16 +16,9 @@ let scheduleData = {
       time: "6:00PM-7:00PM",
     },
     {
-      name: "Swag Drop",
-      description: "Free swag!",
-      tooltip:
-        "Get your free hackathon swag provided by VenusHacks and its sponsors!",
-      time: "6:00PM",
-    },
-    {
-      name: "Team Formation",
+      name: "Teammate Tinder: Team Formation",
       description: "Team up hackers!",
-      tooltip: "Get to know your fellow hackers and find potential teammates!",
+      tooltip: "Stressed about team formation! Speed friendliness is here for you! If you already have a team, you are welcome to join to network with other teams and organizers!",
```

Comment:
minor: `stressed about team formation!` - should that first sentence be a question